app/swagger/swagger/requests/profile.rb

```diff
@@ -1415,6 +1415,113 @@ class Profile
           end
         end
       end
+
+      swagger_path '/v0/profile/contacts' do
+        operation :post do
+          extend Swagger::Responses::AuthenticationError
+          extend Swagger::Responses::ForbiddenError
+
+          key :description, 'Create a Next-of-Kin or Emergency Contact'
+          key :tags, [:profile]
+
+          parameter :authorization
+
+          parameter do
+            key :name, :body
+            key :in, :body
+            key :description, 'Contact attributes'
+            key :required, true
+
+            schema do
+              key :$ref, :Contacts
+            end
+          end
+
+          response 201 do
+            key :description, 'Contact created'
```

Comment:
Should this return the contact ID for reference? I don't think you will use it from the response, but could be useful for debugging.